angucomplete-alt.js

```diff
@@ -298,20 +298,35 @@
         }
       }
 
+      function handleSubmitOnEnter(event) {
+        if (event && !scope.submitOnEnter) {
```

Comment:
hmm... are you sure you tested it with this branch? A change I made to the link function sets `scope.submitOnEnter` to the result of `scope.submitOnEnter === 'true'`, so it should always be a true or false boolean value after the link function has executed.